flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/TemporalTypesTest.scala

```diff
@@ -1158,7 +1157,7 @@ class TemporalTypesTest extends ExpressionTestBase {
     testAllApis(
       toTimestampLtz(100.01.cast(DataTypes.FLOAT()), 0),
       "TO_TIMESTAMP_LTZ(CAST(100.01 AS FLOAT), 0)",
-      "1970-01-01 08:01:40.010")
+      "1970-01-01 08:01:40.000")
```

Comment:
why is it like that?
why are we missing millis now?

Reply:
Hi @snuyanzin, the 010 is the original test case. I reverted it back to 010 and the test passes now. 
I added a precision with 3 in the new Java test.